### PR TITLE
Split pandas

### PR DESCRIPTION
**Pull request by HenryCheval**
changed titles
acknowledges and introduction added
error not fixed

### CODE REVIEW

**Comment by Lola-jo** [comment]
1. Article title not changed
2. Error reported not resolved
3. No output in the second half of the introduction file

**Comment by HenryCheval** [comment]
3 problems solved

File: open-machine-learning-jupyter-book/data-science/working-with-data/pandas/advanced-pandas-techniques.ipynb [hunk 1]
```diff
@@ -0,0 +1,5078 @@
+{
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/pandas/advanced-pandas-techniques.ipynb:1]: <p>Please resolve this error. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/pandas/advanced-pandas-techniques.ipynb:1]: <p>Please resolve this error. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Review by Lola-jo — APPROVED** [review]
ok

**Pull request merged**